lib/network/options.js

```diff
@@ -667,6 +669,8 @@ let configureOptions = {
     selectConnectedEdges: true,
     hoverConnectedEdges: true,
     tooltipDelay: [300, 0, 1000, 25],
+    zoomMin: [0.02, 0, 0.1, 0.005],
+    zoomMax: [10, 9.995, 10, 0.005],
```

Comment:
I don't get this. Why is the range so small?.

Reply:
Mainly because min/max are separate props. But I'm happy to put the range you think is appropriate. What's your suggestion for both props?

Reply:
I'd go with 0 through 10 for both. The input validation should handle cases where max is smaller than min etc.

Reply:
Will do, thank you.